docs/products/kafka/howto/manage-acls.md

```diff
@@ -1,77 +1,472 @@
+- `service_name`: Enter the name of your Aiven for Apache Kafka service.
+- `resource_name`: Specify the resource name or prefix for pattern-based matching.
+- `resource_type`: Specify the type of resource to manage, such as
+  `Topic`, `Group`, `Cluster`, or `TransactionalId`.
+- `pattern_type`: Specify the pattern type for resource matching. Use `LITERAL` for
+  exact matches or `PREFIXED` for pattern-based matching.
+- `principal`: Enter the principal in the format `User:<username>`.
+- `operation`: Specify the Kafka operation, such as `Read`, `Write`, or `Create`.
+- `permission_type`: Specify the type of permission. Use `ALLOW` or `DENY`.
+- `host`: Specify the allowed host, or use `*` to apply to all hosts.
+
+</TabItem>
+</Tabs>
+
+## Add an Aiven ACL entry
+
+<Tabs groupId="acl-methods">
+<TabItem value="console" label="Aiven Console" default>
 
 1. Log in to [Aiven Console](https://console.aiven.io/) and select your
    service.
+1. Click <ConsoleLabel name="acl" />.
+1. Click **Add entry**.
+1. On the **Add access control entry** screen:
+   1. Select **Aiven ACLs** as the ACL type.
+   1. Fill in the following fields:
+      1. **Resource type**: Select `Topic` or `Schema`.
+      1. **Permission type**: Select `admin`, `read`, `write`, or `readwrite`.
+      1. **Username**: Enter the username or pattern to apply the ACL to. Supports
+         wildcards `*` and `?`.
+      1. **Resource**: Enter the name of the topic or schema, or use `*` to apply to all.
+   1. Click **Submit**.
+
+:::tip
+After defining custom ACLs, delete the default `avnadmin` ACL entry by clicking
+<ConsoleLabel name="deleteacl" /> under **Actions**.
+:::
+
+</TabItem>
+<TabItem value="cli" label="Aiven CLI">
+
+To add an Aiven ACL entry with the Aiven CLI, run:
+
+```bash
+avn service acl-add <service_name> \
+  --username <username_pattern> \
+  --permission <permission_type> \
+  --topic <topic_pattern>
+```
+
+Parameters:
+
+- `<service_name>`: Specify the name of your Aiven for Apache Kafka service.
+- `--username`: Specify the username or pattern to apply. Supports wildcards `*` and `?`.
+- `--permission`: Specify the permission type. Valid values are `read`, `write`, or
+  `readwrite`.
+- `--topic`: Specify the topic name or pattern to apply. Supports wildcards `*` and `?`.  .
+
+**Example:**
+
+Allow the username pattern `developer*` to have `read` permissions for all topics
+with names starting with `logs-`:
+
+```bash
+avn service acl-add kafka-service \
+  --username developer* \
+  --permission read \
+  --topic logs-*
+```
+
+</TabItem>
+<TabItem value="api" label="Aiven API">
+
+To add an Aiven ACL entry, use the following API request:
+
+```bash
+curl --request POST \
+  --url https://api.aiven.io/v1/project/<project_name>/service/<service_name>/acl \
+  --header 'Authorization: Bearer <api_token>' \
+  --header 'Content-Type: application/json' \
+  --data '{
+    "username": "<username_pattern>",
+    "resource_type": "<topic|schema>",
```

Comment:
```suggestion
```
There is no such field. Schema related acls are added at separate endpoint `/service/<service_name>/schema-registry/acl`